Sources/TensorFlow/Initializers.swift

```diff
@@ -327,6 +327,37 @@ public extension Tensor where Scalar: Numeric {
     }
 }
 
+public extension Tensor where Scalar: TensorFlowFloatingPoint {
+    /// Creates a 1-D tensor representing a sequence from a starting value, up to and
+    /// including an end value, spaced evenly to generate the number of values specified.
+    ///
+    /// - Parameters:
+    ///   - start: The starting value to use for the sequence. If the sequence contains any values,
+    ///     the first one is `start`.
+    ///   - end: An end value to limit the sequence. `end` is the last element of the resulting
+    ///     sequence.
+    ///   - count: The number of values in the resulting sequence. `count` must be positive.
+    @inlinable
+    init(linSpaceFrom start: Scalar, to end: Scalar, count: Int) {
```

Comment:
Since we're using full names in the API so far, it may be better to rename the argument label to:
```suggestion
    init(linearSpaceFrom start: Scalar, to end: Scalar, count: Int) {
```

Reply:
That makes sense to me. Wasn't sure if we wanted the `linspace` for familiarity when comparing to Numpy and Python TF, but I'm all for descriptive names.